resources/assets/sass/views/_preview.scss

```diff
@@ -46,6 +46,8 @@
 .block-overlay__delete {
 	transition: background-color .3s ease-out;
 	position: absolute;
+	right: 50px !important;
```

Comment:
What's the reason for using important here? I don't think we should ever use it... ever haha.
If the specificity of the CSS is too low, can we increase it in a less gross way?